sssom/parsers.py

```diff
@@ -862,8 +862,8 @@ def split_dataframe(
     """
     if msdf.df is None:
         raise RuntimeError
-    subject_prefixes = set(msdf.df[SUBJECT_ID].str.split(":", 1, expand=True)[0])
-    object_prefixes = set(msdf.df[OBJECT_ID].str.split(":", 1, expand=True)[0])
+    subject_prefixes = set(msdf.df[SUBJECT_ID].str.split(":", n=1, expand=True)[0])
+    object_prefixes = set(msdf.df[OBJECT_ID].str.split(":", n=1, expand=True)[0])
```

Comment:
minor edit ... [explanation](https://stackoverflow.com/questions/73886976/facing-error-in-pandas-all-arguments-of-stringmethods-split)